Documentation improvement (#101)
lxylxy123456/ngshare: docs/contributer_guide/system_architecture.rst

System Architecture Overview
============================

> ``ngshare`` is intended to run as a Kubernetes pod and service outside JupyterHub. It will still be proxied by JupyterHub's proxy service, and can be accessed from any JupyterHub user pod. It uses the Hub for authentication.

Reviewer: ```suggestion
``ngshare`` is intended to run as a Kubernetes pod and service outside JupyterHub. In a Kubernetes setup, ngshare is proxied by JupyterHub's proxy service, and can be accessed from any JupyterHub user pod. It uses the Hub for authentication.
```
